manticore/models.py

```diff
@@ -0,0 +1,80 @@
+from .utils.helpers import issymbolic
+from .core.smtlib.solver import solver
+from .core.smtlib.operators import ITEBV, ZEXTEND
+
+
+def _find_zero(cpu, constrs, ptr):
```

Comment:
i'm not sure i want to make models.py a subpackage just to include a helpers file. keep it as is and change later when models.py gets messy or an external user needs _find_zero?